composables/timeline.ts

```diff
@@ -11,7 +11,7 @@ function areStatusesConsecutive(a: mastodon.v1.Status, b: mastodon.v1.Status) {
 
 function removeFilteredItems(items: mastodon.v1.Status[], context: mastodon.v1.FilterContext): mastodon.v1.Status[] {
   const isStrict = (filter: mastodon.v1.FilterResult) => filter.filter.filterAction === 'hide' && filter.filter.context.includes(context)
-  const isFiltered = (item: mastodon.v1.Status) => !item.filtered?.find(isStrict)
+  const isFiltered = (item: mastodon.v1.Status) => (item.account && item.account.id === currentUser.value?.account.id) || !item.filtered?.find(isStrict)
```

Comment:
You may be able to remove this extra condition by updating the test if `item.account` always is defined.

Reply:
Yes, should always be defined. I updated the fake Statuses in our unit tests to always have the account :)